contentcuration/contentcuration/static/js/edit_channel/tree_edit/hbtemplates/container_area.handlebars

```diff
@@ -33,19 +42,43 @@
 			<a class="nav-label archive_button disable-on-edit">Trash</a>
 		</div>
 		{{/if}}
+		{{#unless staging}}
 		<div class="nav-button-container">
 			<div class="nav-button permissions_button disable-on-edit"><span class="glyphicon glyphicon-user"></span></div>
 			<a class="nav-label permissions_button disable-on-edit">Invite</a>
 		</div>
+		{{/unless}}
+		{{#if channel.ricecooker_version}}
+			{{#unless view_only}}
```

Comment:
is it possible to combine `view_only` and `staging` inside this `{{#unless}}`? Logically, `!view_only && !staging` is equivalent to `!(view_only || staging)` or `{{#unless view_only || staging}}` if that syntax is allowed in handlebars.